release/restcomm-assembly-as7/src/main/resources/descriptor_all-in-one.xml

```diff
@@ -0,0 +1,68 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+    <id>assembly</id>
```

Comment:
this is the "right" place i think. this id is same as standalone one, and could create potential collision

Reply:
this should be addressed with https://github.com/RestComm/Restcomm-Connect/pull/2937/commits/71fd01f49b9dc91b099c0aaa1cbf9720fe5fb5ee